packages/wallet-ui/src/utils/utils.ts

```diff
@@ -79,6 +93,21 @@ export const getHumanReadableAmount = (
   return amountStr.substring(0, indexDecimal + firstNonZeroIndex + 3);
 };
 
+export const getSpendableTotalBalance = (
+  asset: Erc20TokenBalance,
+  assetAmount?: string,
+): string => {
+  const spendableAmount = getHumanReadableAmount(
+    asset,
+    assetAmount,
+    BalanceType.Spendable,
+  );
+  const totalAmount = getHumanReadableAmount(asset);
+  return spendableAmount === totalAmount
+    ? `${spendableAmount}`
+    : `${spendableAmount} (${totalAmount})`;
+};
+
```

Comment:
as`getSpendableTotalBalance` is only used for a case to display the balance, hence the assetAmount doesnt really needed

```suggestion
export const getSpendableTotalBalance = (
  asset: Erc20TokenBalance
): string => {
  const spendableAmount = getHumanReadableAmount(
    asset,
    asset.spendableAmount
  );

  if (asset.spendableAmount === asset.total) {
      return `${spendableAmount}`
  }
  
  const totalAmount = getHumanReadableAmount(
    asset,
    asset.amount
  );
  
   return `${spendableAmount} (${totalAmount})`;
};

```